utils/preDeployContracts.js

```diff
@@ -0,0 +1,53 @@
+const { deployContracts } = require('./deployContracts');
+
+const fs = require('fs-extra');
+
+module.exports = {
```

Comment:
Since there are already deep levels of nesting, it might be clearer to define
```
function preDeployContracts(
  buildContractsPath,
  appPreBuiltContractArtifactsPath,
)
```
and then add the function to the `exports` object `module.exports = {preDeployContracts}`

Reply:
:+1: